source/manual/bouncer.html.md

```diff
@@ -0,0 +1,89 @@
+---
+owner_slack: '#navigation'
+review_by: 2017-07-26
+title: Bouncer
+section: Transition
+layout: manual_layout
+parent: "/manual.html"
+old_path_in_opsmanual: "../opsmanual/2nd-line/applications/bouncer-and-transition.html"
+---
+
+# Bouncer
+
+[Bouncer](https://github.com/alphagov/bouncer) is a Ruby/Rack web app that receives requests for the old URLs,
```

Comment:
I'm not sure it's clear what 'old URLs' refers to here – can this be a little more specific?